grupo-gomez-lopez/src/redux/Product/thunks.js

```diff
@@ -0,0 +1,24 @@
+import {
+  setProducts,
+  showProducts,
+  dismissIsLoading,
+  setIsLoading,
+} from "./actions";
+
+export const getProducts = async (dispatch) => {
+  dispatch(setIsLoading());
+  let response = await fetch(
+    "https://mcga-2022-backend.vercel.app/api/products"
+  );
+  //let response = await fetch("http://localhost:3000/api/products");
+  let data = await response.json();
+  if (data) {
+    dispatch(setProducts(data.data));
+    dispatch(dismissIsLoading());
+  }
+  if (data.status === 400) {
```

Comment:
you are only handling one case of error you should wrap your entire inside function in a try catch statement, and check status !== 200 instead of === 400